src/App.js

```diff
@@ -1,25 +1,79 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import contacts from "./contacts.json";
+import { useState } from "react";
+
+const fiveContacts = contacts.slice(0, 5);
 
 function App() {
+  const [contact, setContact] = useState(fiveContacts);
+
+  const newContacts = contacts.slice(6);
+
+  const randomContact = () => {
```

Comment:
If you want to get a random contact that is not listed, you should apply the Math random on newContacts array, not on the original one.